### ISSUE TITLE
about    run  app.py (Segment-and-track-anything)  in   long video(images>100),result  huav error 

### ISSUE THREAD

**Issue by zhanghongyong123456** [qiuyu96/CoDeF#39]
![image](https://github.com/qiuyu96/CoDeF/assets/48466610/4684aa0a-0012-45a9-9aee-1d25b39f82d4)
![image](https://github.com/qiuyu96/CoDeF/assets/48466610/b9654c76-a386-465e-b2ff-85e5e34a7080)

**Comment by zhanghongyong123456** [comment]
when i set sam_gap > my video img num，and  Reset  button  update,it's OK,but output masks
Several more masks (*new.png)，need  remove   *new.png

**Comment by zhanghongyong123456** [comment]
> How about setting `minimum = 1` [here](https://github.com/z-x-yang/Segment-and-Track-Anything/blob/main/app.py#L586) ?

The same result, can be tracked correctly, but the masks are still a few more  masks (*new.png)，need remove *new.png

**Comment by LiNO3Dy** [comment]
The model automatically detects and adds new objects every sam_gap frames. To solve this issue, you can adjust the default value of **sam_gap** in SegTrackerArgs to **its maximum value** and **click reset**.

**Comment by zhanghongyong123456** [comment]
> The model automatically detects and adds new objects every sam_gap frames. To solve this issue, you can adjust the default value of **sam_gap** in SegTrackerArgs to **its maximum value** and **click reset**.

I  test  set  sam_gap    = 9999， The same result, can be tracked correctly, but the masks are still a few more masks (*new.png)，
![image](https://github.com/qiuyu96/CoDeF/assets/48466610/020d4abf-c906-40de-8569-04febbefad6c)

**Comment by zhanghongyong123456** [comment]
> How about setting `minimum = 1` [here](https://github.com/z-x-yang/Segment-and-Track-Anything/blob/main/app.py#L586) ?

The results of my training are not very good, is it because I exercise too fast?,
my  video  fps=30， base.yaml  fps=15, need set 5 or 1 it is OK?
![image](https://github.com/qiuyu96/CoDeF/assets/48466610/a86f9892-c8dd-4132-a2dc-2cc502edde91)
![image](https://github.com/qiuyu96/CoDeF/assets/48466610/ca7374e7-62e2-4cbc-a75c-07ebea443e31)
![image](https://github.com/qiuyu96/CoDeF/assets/48466610/0a95297f-88d5-444a-a6ea-ca375caa8ee8)

Setting those hyperparameters can improve the reconstruction effect of the model

**Comment by ken-ouyang** [comment]
@zhanghongyong123456 Yes. The motion cannot be so fast as (x,y,t) is viewed as a 3D volume.

**Comment by zhanghongyong123456** [comment]
May I ask if you will publish a document about training hyperparameter settings. I found that many people are asking about long videos and fast exercise training skills. I saw your reply, but we don’t know how to set them up.
